config/manager/kustomization.yaml

```diff
@@ -18,5 +18,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/kubevirt/ssp-operator
-  newTag: latest
+  newName: quay.io/borod108/ssp-operator
+  newTag: devel
```

Comment:
Committed by mistake.

Reply:
committed very purposefully, but removing now as going out of wip :)